pkgs/by-name/wl/wl-gammarelay-rs/package.nix

```
@@ -0,0 +1,30 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  stdenv,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "wl-gammarelay-rs";
+  version = "0.3.2";
+
+  src = fetchFromGitHub {
+    owner = "maxverevkin";
+    repo = "wl-gammarelay-rs";
+    rev = "v${version}";
+    hash = "sha256-md6e9nRCs6TZarwFD3/GQEoJSIhtdq++rIZTP7Vl0wQ=";
+  };
+
+  cargoHash = "sha256-TDP5RC7B7/ldpK22WsmXd6fSl2rHtvG0hP9NYzoEVYo=";
+
+  nativeBuildInput = [stdenv.cc];
+
+  meta = {
+    description = "A simple program that provides DBus interface to control display temperature and brightness under wayland without flickering.";
```

https://github.com/NixOS/nixpkgs/blob/8f4970162be52371060978d7436451ea77a24f28/pkgs/README.md#meta-attributes
```suggestion
    description = "A simple program that provides DBus interface to control display temperature and brightness under wayland without flickering";
```